portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,4 +93,21 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /** Returns the integer entered by user on how many comments  */
+  private int getNumDisplayComments(HttpServletRequest request) {
+    // Get the input from the form on MaxComments and convert to integer to the
+    // toGet function.
+    String CommentNumString = request.getParameter("max-comments");
```

Comment:
Common Java style is that local variables should be in lowerCamelCase:

```suggestion
    String commentNumString = request.getParameter("max-comments");
```